content/blog/2022/01/2022-01-14-happy-new-year.adoc

```diff
@@ -0,0 +1,254 @@
+Removals were accompanied by plugin updates as needed to retain compatibility and functionality.
+
+// Upgrades
+Key libraries were updated to use more recent releases of the libraries.
+Guava was upgraded from 11.0.1 to 31.0.1.
+Guice was upgraded from 4.0 to 5.0.1.
+Groovy was upgraded from 2.4.12 to 2.4.21.
+Many Apache Commons libraries were upgraded to their most recent releases.
+
+=== Continuous delivery for plugins
+
+Continuous delivery of Jenkins components was proposed in 2020 by Jesse Glick as link:https://github.com/jenkinsci/jep/blob/master/jep/229/README.adoc[Jenkins Enhancement Proposal 229].
+By the end of 2021, 119 plugins had adopted continuous delivery, providing new plugin releases each time a relevant commit was merged to the plugin repository.
+Additional components have adopted continuous delivery as well, including the plugin bill of materials and the Jenkins test harness.
+
+We look forward to even greater adoption of continuous delivery for plugins in 2022.
+
+=== Prefer Java 11 instead of Java 8
+
+Java 11 was adopted as the recommended JDK during 2021.
+Docker images now use JDK 11 by default.
+See the link:/blog/2021/08/17/docker-images-use-jdk-11-by-default/[blogpost] for more information abou the Docker image transition.
+
+== More inclusive naming
+
+The Jenkins project decided in 2016 to replace the term "slave" with the more inclusive term "agent".
+In July 2020 the project adopted the "controller" term to replace the older term "master".
+
+Jenkins core 2.319.1 was released in December 2021 replaced the term "master" with more accurate terminology.
+The release also includes an integrated migration tool to allow existing installations to decide when they would adopt the new terminology.
+
+== Security improvements
+
+Jenkins security improvements have continued throughout 2021.
+The Jenkins security team provided timely responses to security issues in Jenkins core and in Jenkins plugins.
+The project is sincerely grateful to link:/blog/authors/daniel-beck/[Daniel Beck] for his years of service as Jenkins Security Officer.
+link:/blog/authors/wadeck/[Wadeck Follonier] began his service as Jenkins Security Officer in December, 2021.
+
+The Jenkins infrastructure team resolved infrastructure issues and safeguarded Jenkins infrastructure.
+The project is deeply grateful to link:/blog/authors/olblak/[Olivier Vernin] for his years of service as Jenkins Infrastructure Officer.
+link:/blog/authors/dduportal/[Damien Duportal] began his service as Jenkins Infrastructure Officer in December, 2021.
+
+=== Agent to controller security
+
+Daniel Beck proposed link:https://github.com/jenkinsci/jep/tree/master/jep/235[Jenkins Enhancement Proposal 235] in November, 2021 to remove the ability to disable or customize the agent-to-controller security system.
```

Comment:
Oh wow I had no idea this was done. Someone on irc pointed out https://www.jenkins.io/redirect/security-144 is recommending people access a UI that no longer exists, so probably should get removed (separate issue)

Reply:
cc @daniel-beck ^^

Reply:
I think we'll want to modify https://www.jenkins.io/redirect/security-144 rather than remove it, since the current LTS, Jenkins 2.319.2, still provides that functionality.